packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -660,7 +660,18 @@ class _UiKitViewState extends State<UiKitView> {
   }
 
   void _onFocusChange(bool isFocused) {
-    // TODO(hellohuanlin): send 'TextInput.setPlatformViewClient' channel message to engine after the engine is updated to handle this message.
+    if (!isFocused) {
+      // Unlike Android, we do not need to send "clearFocus" channel message
+      // to the engine, because focusing on another view will automatically
+      // cancel the focus on the previously focused platform view.
+      return;
+    }
+    SystemChannels.textInput.invokeMethod<void>(
+      'TextInput.setPlatformViewClient',
+      // _controller must not be nil because _controller and _focusNode are
+      // set at the same time, and this function is triggered by _focusNode.
+      <String, dynamic>{'platformViewId': _controller!.id},
```

Comment:
There could be some change in the way how things are initialized and making `_controller!` crash, right? It is hard to catch because it happens in method channel. 

For example, if someone moved the `_focusNode = FocusNode(debugLabel: 'UiKitView(id: $id)');` out of `_createNewUiKitView`, it isn't obvious that this will crash.

I wonder if we should add an `assert(_controller!=null) in the beginning of this method and then add a:
```dart
if (!_controller) {
  printWarning('warning: attempt to focus a UIKitView when it is not ready. ');
  return;
}
```

Reply:
good point

Reply:
My suggestion above means it won't be null at this point.

Also if we're deviating from the other patterns in this file, prefer `Object` or `Object?` to `dynamic` to opt into type safety.